.github/workflows/kind.yml

```diff
@@ -19,6 +19,11 @@ jobs:
       - name: Checkout the code
         uses: actions/checkout@v2
 
+      - name: Setup oc
```

Comment:
Can we have this part of Makefile? We tried to have Makefile as reusable as possible, so even all dependecies are isntalled in Makefile

Reply:
I believe this approach may not be the best for supporting different operating systems and environments. Requiring the oc command to be pre-installed seems like a reasonable approach, similar to how we handle kubectl.
However, if you still think it's necessary to install oc, I can add an installation step for Linux systems in case oc isn’t already installed.
Additionally, I’m adding an environment variable OC to allow flexibility in changing the executor path if needed.

Reply:
I will change this in a different PR